hip-service/Link/Patient/Communication.cs

```diff
@@ -0,0 +1,16 @@
+using HipLibrary.Patient.Models;
+
+namespace hip_service.OTP
+{
+    public class Communication
+    {
+        public IdentifierType Mode { get; set; }
```

Comment:
What makes you the name the enum `IdentifierType`?

Reply:
Created a enum named CommunicationMode